package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "eslint.github.io",
+  "version": "1.0.0",
+  "description": "This contains the code running on eslint.org.",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "build": "webpack --module-bind json js/app/parser/index.js js/app/parser/index.built.js",
```

Comment:
As I said, I believe you can just do `require('espree')`. Here's how the file is created: https://github.com/eslint/eslint/blob/master/Makefile.js#L561